projects/rero/ng-core/src/lib/pipe/callback.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'callback'
```

Comment:
I think "callback" is too generalistic, maybe callbackArrayFilter or something like that would be better. But the way to do it is very clean